HB/structure.elpi

```diff
@@ -460,6 +514,7 @@ declare-class+structure MLwP Sort
 
   w-params.then MLwP (mk-parameter explicit) (mk-parameter explicit)
     synthesize-fields.body ClassDeclaration,
+  coq.say "ClassDeclaration" ClassDeclaration,
```

Comment:
```suggestion
  if-verbose (coq.say {header} "ClassDeclaration" ClassDeclaration),
```

Reply:
Oh, I think this is a remnant of my debug prints. I removed it.